plugins/WebsiteMeasurable/MeasurableSettings.php

```diff
@@ -84,6 +85,10 @@ public function __construct(SitesManager\API $api, Plugin\Manager $pluginManager
 
     protected function init()
     {
+        if ($this->idMeasurableType !== Type::ID && $this->idMeasurableType !== MobileAppType::ID) {
+            return;
+        }
+
```

Comment:
Logically that would be right. But I'm wondering if there would be a better way to switch on/off the settings a measurable type defines. If for example someone wants to create another custom measurable type that should be based on the website settings. Could we maybe check the inheritance of the Type class or something like that and let the MobileType class inherit from the WebsiteType class?